src/test/java/uk/gov/ons/ctp/response/casesvc/CaseCreator.java

```diff
@@ -44,7 +44,7 @@ public CaseNotification sendSampleUnit(
     SampleUnitParent sampleUnit = new SampleUnitParent();
     sampleUnit.setCollectionExerciseId(collectionExerciseId.toString());
     sampleUnit.setId(sampleUnitId.toString());
-    sampleUnit.setActionPlanId(UUID.randomUUID().toString());
+    //sampleUnit.setActionPlanId(UUID.randomUUID().toString());
```

Comment:
Remove if not needed

Reply:
Sure.. But If I remove it I can't mark the test config deprecated as true. I will uncomment it for now.

Reply:
👍